### PR TITLE
cmd/bosun: Tweak dashboard panels to be more compact

### PR DESCRIPTION
**Pull request by kylebrandt**

### CODE REVIEW

File: cmd/bosun/web/static/partials/alertstate.html [hunk 1]
```diff
@@ -13,77 +13,88 @@ <h4 class="panel-title">
...
-		</div>
-		<div class="row">
-			<div class="col-sm-3 text-right"><strong>History</strong></div>
-			<div class="col-sm-9">
-				<a ng-href="/history?key={{encode(name)}}" ng-bind="name"></a>
+	<div class="row">
+		<div class="col-sm-6">
+			<div class="row">
+				<div class="col-sm-3 text-right">
+					<p>
```
Comment (captncraig) [inline cmd/bosun/web/static/partials/alertstate.html]: can we condense the spacing here?

File: cmd/bosun/web/static/partials/alertstate.html [hunk 2]
```diff
@@ -13,77 +13,88 @@ <h4 class="panel-title">
...
 			</div>
-		</div>
-		<div class="row">
-			<div class="col-sm-3 text-right"><strong>Rule</strong></div>
-			<div class="col-sm-9">
-				<a ng-href="{{state.RuleUrl}}" ng-bind="name"></a>
+			<div class="row">
+				<div class="col-sm-3 text-right">
+					<p>
+						<strong class="text-right">State:</strong>
```
Comment (captncraig) [inline cmd/bosun/web/static/partials/alertstate.html]: one line all this

File: cmd/bosun/web/static/partials/alertstate.html [hunk 3]
```diff
@@ -13,77 +13,88 @@ <h4 class="panel-title">
...
-		</div>
-		
-		<div class="row">
-			<div class="col-sm-3 text-right"><strong>Expr</strong></div>
-			<div class="col-sm-9">
-				<a ng-href="/expr?expr={{btoa(state.Expr)}}" ng-bind="zws(state.Expr)"></a>
+			<div class="row">
+				<div class="col-sm-3 text-right">
+					<p>
+						<strong class="text-right">Views:</strong>
```
Comment (captncraig) [inline cmd/bosun/web/static/partials/alertstate.html]: and here

Comment (kylebrandt) [inline cmd/bosun/web/static/partials/alertstate.html]: Yes it gets the spacing right, but will condense the formatting

**Comment by captncraig** [comment]
Other than formatting and whitespace LGTM

File: cmd/bosun/web/static/partials/alertstate.html (hunk 3 above)
Comment (mjibson) [inline cmd/bosun/web/static/partials/alertstate.html]: I prefer the multi line format myself.

**Pull request merged**